From d8952b2dfeb31d6c88c5d8f29b786393876083e9 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:11 -0400
Subject: [redhat] kabi: Adding symbol netdev_rx_handler_unregister (net/core/dev.c)

Message-id: <e64b582e87b968c654ff505a86763c813a0cc921.1410510773.git.poros@redhat.com>
Patchwork-id: 95944
O-Subject: [PATCH RHEL7.1 131/233] kabi: Adding symbol netdev_rx_handler_unregister (net/core/dev.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    To unregister n1kv handlers during module unload time (Done as part of package Upgrade).

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/netdev_rx_handler_unregister
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/netdev_rx_handler_unregister
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/netdev_rx_handler_unregister

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/netdev_rx_handler_unregister  b/redhat/kabi/kabi-module/kabi_ppc64/netdev_rx_handler_unregister 
new file mode 100644
index 0000000..3b1f947
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/netdev_rx_handler_unregister  
@@ -0,0 +1,2 @@
+#1-
+0xbe6884f9 netdev_rx_handler_unregister vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/netdev_rx_handler_unregister  b/redhat/kabi/kabi-module/kabi_s390x/netdev_rx_handler_unregister 
new file mode 100644
index 0000000..5dbd45d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/netdev_rx_handler_unregister  
@@ -0,0 +1,2 @@
+#1-
+0x3ff7a505 netdev_rx_handler_unregister vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/netdev_rx_handler_unregister  b/redhat/kabi/kabi-module/kabi_x86_64/netdev_rx_handler_unregister 
new file mode 100644
index 0000000..c99fa5c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/netdev_rx_handler_unregister  
@@ -0,0 +1,2 @@
+#1-
+0xb0d91372 netdev_rx_handler_unregister vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1